python-mip/tests/unit/fixtures.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+import numpy as np
+
+
+def metadata():
+    # see https://github.com/HBPMedical/woken/blob/master/src/test/resources/metadata/mip_cde_variables.json for
+    # all definitions
+    return {
+        'lefthippocampus': {
+            'code': 'lefthippocampus',
+            'type': 'real',
+            'mean': 3.0,
+            'std': 0.35
+        },
+        'minimentalstate': {
```

Comment:
@ludovicc your branch assumes the following format of META definition
```
{
'name': 'lefthippocampus',
'type': {'name': 'real'},
'mean': 3.0,
'std': 0.35
}
```
shouldn't it be rather
```
'name': 'lefthippocampus',
'type': {
    'name': 'real',
    'mean': 3.0,
    'std': 0.35
}
```
to be consistent with nominal types that return
```
{
        'name': 'agegroup',
        'type': {
            'name': 'polynominal',
            'enumeration': ['-50y', '50-59y', '60-69y', '70-79y', '+80y']
        },
        'series': ['70-79y', '70-79y', '70-79y', '70-79y', '+80y']
    }
```
? Thanks!